### PR TITLE
Add

### PR DESCRIPTION
**Pull request by coderp233**
Added the Sign up Admin page, scss styling and enable button

### CODE REVIEW

File: src/Components/SignupAdmin/SignupAdmin.scss
Comment (hereisadi) [inline src/Components/SignupAdmin/SignupAdmin.scss:1]: use relative units

File: src/App.jsx
Comment (hereisadi) [inline src/App.jsx:1]: solve all ESLint issues in all files

**Review by hereisadi — CHANGES_REQUESTED** [review]

File: src/Components/SignupAdmin/SignupAdmin.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import './SignupAdmin.scss'
+
+
+const SignupAdmin=()=>{
+
+    return(
+        <div className="admin-form">
```
Comment (hereisadi) [inline src/Components/SignupAdmin/SignupAdmin.jsx:8]: you have to use className like:
className = {styles.admin_form}

File: src/Components/SignupAdmin/SignupAdmin.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import './SignupAdmin.scss'
```
Comment (hereisadi) [inline src/Components/SignupAdmin/SignupAdmin.jsx:2]: import like:
import styles from './Signup.module.scss'

File: src/Components/SignupAdmin/SignupAdmin.scss
Comment (hereisadi) [inline src/Components/SignupAdmin/SignupAdmin.scss:1]: create scss file like:
SignupAdmin.module.scss

File: src/Components/SignupAdmin/SignupAdmin.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      </div>
+      <br/>
+      <div className="input-group">
+        <input id='control1' type="email" className="input" required="required" />
+        <label for="control1" className="Placeholder">Email</label>
+      </div>
+      
+      <div className="input-group" >
+      <label for="Designation1" className="input" placeholder="Designation"></label>
+  <select id='Designation1' name="design" className="select_step" style={{width:'326px', height:'40px',position: "relative", top: "-22px"}} >
```
Comment (hereisadi) [inline src/Components/SignupAdmin/SignupAdmin.jsx:25]: use relative units. for eg: `rem`, `vw`, `vh` etc

**Review by hereisadi — CHANGES_REQUESTED** [review]

File: src/Components/SignupAdmin/SignupAdmin.jsx [hunk 4]
```diff
@@ -8,43 +8,49 @@ const SignupAdmin=()=>{
...
+        <input type="text" required="required" className={styles.input} id="control" placeholder='Name'/>
+        <label className={styles.Placeholder} for="control">Name</label>
       </div>
-      <br/>
+    
       <div className={styles.input_group}>
-        <input id='control1' type="email" className={styles.input} required="required"  />
-        <label htmlFor="control1" className={styles.Placeholder}>Email</label>
+        <input id='control1' type="email" className={styles.input} required="required" placeholder='Email' />
+        <label for="control1" className={styles.Placeholder}>Email</label>
```
Comment (Himangshu-101) [inline src/Components/SignupAdmin/SignupAdmin.jsx:22]: use htmlFor only for ReactJS. 'for' is a reserved keyword in JS
Use this in all occurring cases in this or other files too

File: src/Components/SignupAdmin/SignupAdmin.jsx [hunk 5]
```diff
@@ -8,43 +8,49 @@ const SignupAdmin=()=>{
...
+        
+      {/* <p  className={styles.input} placeholder="Designation"></p> */}
+  <select id='control5' name="design" placeholder='select one-' className={styles.select_step} style={{width:'20.375rem', height:'5.5vh'}} >
+    <option value="select one">select one</option>
+  <option value="supervisor" >Supervisor</option>
+  <option value="warden" >Warden</option>
+  <option value="dean"  >Dean</option>
+
 </select>
+<label For="control5" >Designation</label>
```
Comment (Himangshu-101) [inline src/Components/SignupAdmin/SignupAdmin.jsx:35]: JS, ReactJS are case sensitive, 'For' is not a word in either.
Use 'htmlFor' here

File: src/Components/SignupAdmin/SignupAdmin.jsx [hunk 6]
```diff
@@ -8,43 +8,49 @@ const SignupAdmin=()=>{
...
+        <input id="control3" type="password" className={styles.input} required="required" placeholder='password'/>
+        <label for="control3" className={styles.Placeholder}>Password</label>
       </div>
-      <br/>
+      
       <div className={styles.input_group}>
-        <input id="control4" type="password" className={styles.input} required="required" />
-        <label htmlFor="control4" className={styles.Placeholder}>Confirm Password</label>
+        <input id="control4" type="password" className={styles.input} required="required" placeholder='confirm'/>
+        <label id="control4" className={styles.Placeholder}>Confirm Password</label>
```
Comment (Himangshu-101) [inline src/Components/SignupAdmin/SignupAdmin.jsx:50]: add htmlFor

**Review by Himangshu-101 — CHANGES_REQUESTED** [review]

**Review by ankit07an — APPROVED** [review]

**Pull request closed without merging**